Python/bytecodes.c

```diff
@@ -3302,7 +3324,7 @@ dummy_func(
             top = Py_NewRef(bottom);
         }
 
-        inst(BINARY_OP, (unused/1, lhs, rhs -- res)) {
+        op(_BINARY_OP_SPECIALIZE, (unused/1, lhs, rhs -- lhs, rhs)) {
```

Comment:
Good point. In tier 2 we have to set `ENABLE_SPECIALIZATION` to false, since specializing a tier 2 instruction makes no sense -- that's the job of tier 1.